_security/configuration/index.md

```diff
@@ -14,18 +14,74 @@ redirect_from:
 The plugin includes demo certificates so that you can get up and running quickly. To use OpenSearch in a production environment, you must configure it manually:
 
 1. [Replace the demo certificates]({{site.url}}{{site.baseurl}}/install-and-configure/install-opensearch/docker/#configuring-basic-security-settings).
-1. [Reconfigure `opensearch.yml` to use your certificates]({{site.url}}{{site.baseurl}}/security/configuration/tls).
-1. [Reconfigure `config.yml` to use your authentication backend]({{site.url}}{{site.baseurl}}/security/configuration/configuration/) (if you don't plan to use the internal user database).
-1. [Modify the configuration YAML files]({{site.url}}{{site.baseurl}}/security/configuration/yaml).
-1. If you plan to use the internal user database, [set a password policy in `opensearch.yml`]({{site.url}}{{site.baseurl}}/security/configuration/yaml/#opensearchyml).
-1. [Apply changes using the `securityadmin` script]({{site.url}}{{site.baseurl}}/security/configuration/security-admin).
-1. Start OpenSearch.
-1. [Add users, roles, role mappings, and tenants]({{site.url}}{{site.baseurl}}/security/access-control/index/).
+- OpenSearch ships with demo certificates intended for quick setup and testing purposes. For a production environment, it's critical to replace these with your own trusted certificates to ensure secure communication.
+  
+- Steps:
+  - Generate your own certificates: Use tools like OpenSSL or a certificate authority (CA) to generate your own certificates.
+  - Place generated certificates and private key to the appropriate OpenSearch directory.
+  - Update file permissions: Ensure the files are accessible by the OpenSearch service user and have proper permissions set to avoid unauthorized access.
+
+2. [Reconfigure `opensearch.yml` to use your certificates]({{site.url}}{{site.baseurl}}/security/configuration/tls). 
+- The `opensearch.yml` file is the main configuration file for OpenSearch. Update this file to point to your custom certificates for secure communication.
+- Steps:
+  - Open `opensearch.yml`: Locate and open the `opensearch.yml` file in your preferred text editor.
+  - Update paths to certificates and keys: Set the correct paths for your certificates and keys in the `opensearch.yml` file. For example:
+```json
+plugins.security.ssl.transport.pemcert_filepath: /path/to/your/cert.pem
+plugins.security.ssl.transport.pemkey_filepath: /path/to/your/key.pem
+plugins.security.ssl.transport.pemtrustedcas_filepath: /path/to/your/ca.pem
+plugins.security.ssl.http.enabled: true
+plugins.security.ssl.http.pemcert_filepath: /path/to/your/cert.pem
+plugins.security.ssl.http.pemkey_filepath: /path/to/your/key.pem
+plugins.security.ssl.http.pemtrustedcas_filepath: /path/to/your/ca.pem
+```
+3. [Reconfigure `config.yml` to use your authentication backend]({{site.url}}{{site.baseurl}}/security/configuration/configuration/) (if you don't plan to use the internal user database).
```

Comment:
nit: remove `(if you don't plan to use the internal user database).`

Users will always need to configured at least 1 authc backend. If they are using the demo configuration then the internal users is already enabled for demonstration purposes.